src/main/java/frc/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -521,8 +524,9 @@ public void simulationPeriodic() {
           intakeState = IntakeState.LOCATE_CORAL;
           break;
         case ALGAE_INTAKE:
-          Commands.waitSeconds(0.5);
-          intakeState = IntakeState.ALGAE_HOLD;
+          if (sensors.getSensorBool(kSensor5ID)) {
+            intakeState = IntakeState.ALGAE_HOLD;
+          }
```

Comment:
This will always return the default value of the sensor. For this to work in simulation there needs to be logic to set the sensor's simulation value like setSimState does for the CoralEnum state.